things/index.html

```diff
@@ -100,7 +102,8 @@ <h3>Logos, Graphics and Animations</h3>
           </div>
         </div>
       </div>
-
+<!-- Just a design thing, but the image for mystery sistery feels different from 
```

Comment:
I agree. When I went in and added these images I meant to change that, but just never did. Thank you for reminding me.